app.js

```diff
@@ -20,20 +10,47 @@ You are running the app using production config (Firestore Database, Map API key
 
 If you are developing the app locally, please use "npm run dev" to start the app.
 `;
+
+// Load API keys.
+require('dotenv').config();
+
+// Show production warning message.
+if (process.env.NODE_ENV === 'production') {
+  console.log(PROD_WARNING_MESSAGE);
+}
+
+// Module dependencies.
+var express = require('express');
+var http = require('http');
+var path = require('path');
+var handlebars = require('express-handlebars')
+var passport = require('passport');
+var cookieSession = require('cookie-session');
```

Comment:
I think all of these ```var```s can be ```const```s.

Reply:
This will be fixed in the auto-format PR.